sale_commission_pricelist/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==============================
+Sales commissions by pricelist
+==============================
+
+This module extends the functionality of sale_commission to allow you set a
+commission to pricelist item.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to pricelist
```

Comment:
- Go to Sales -> Configuration -> Settings -> Pricing -> Sale Price and activate: "Advanced pricing based on formulas (discounts, margins, rounding)"
- Go to Sales -> Configuration -> Price-list and edit an existing one or create a new one
- In the price-list -> add a new item or open a new one
- You will find the field commission in the price-list item